lab-lindsay/app/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html ng-app="cowsayApp">
```

Comment:
if you were to place ng-app on any other element, what would it do? can you do that? and what are the implications if you could?